docs/storage/multipathing.md

```diff
@@ -0,0 +1,262 @@
+    end
+    lun1@{ shape: lin-cyl, label: "LUN" }
+
+  end
+
+  card1-pif1-ip([10.42.1.11/24]):::blueNode
+  card1-pif2-ip([other networks]):::grayNode
+  card2-pif2-ip([10.42.2.11/24]):::greenNode
+  card2-pif1-ip([other networks]):::grayNode
+  ctrl1-port1-ip([10.42.1.101/24]):::blueNode
+  ctrl1-port2-ip([10.42.2.101/24]):::greenNode
+  ctrl2-port1-ip([10.42.1.102/24]):::blueNode
+  ctrl2-port2-ip([10.42.2.102/24]):::greenNode
+
+  card1pif1<-->card1-pif1-ip
+  card1-pif1-ip<-->vlan421
+  vlan421<-->ctrl1-port1-ip
+  ctrl1-port1-ip<-->ctrl1-port1
+  vlan421<-->ctrl2-port1-ip
+  ctrl2-port1-ip<-->ctrl2-port1
+
+  card2pif2<-->card2-pif2-ip
+  card2-pif2-ip<-->vlan422
+  vlan422<-->ctrl1-port2-ip
+  ctrl1-port2-ip<-->ctrl1-port2
+  vlan422<-->ctrl2-port2-ip
+  ctrl2-port2-ip<-->ctrl2-port2
+
+  card1pif2<-->card1-pif2-ip
+  card2pif1<-->card2-pif1-ip
+  ctrl1 <----> lun1
+  ctrl2 <----> lun1
+
+linkStyle 0 stroke:#4A90E2,stroke-width:2px;
+linkStyle 1 stroke:#4A90E2,stroke-width:2px;
+linkStyle 2 stroke:#4A90E2,stroke-width:2px;
+linkStyle 3 stroke:#4A90E2,stroke-width:2px;
+linkStyle 4 stroke:#4A90E2,stroke-width:2px;
+linkStyle 5 stroke:#4A90E2,stroke-width:2px;
+
+linkStyle 6 stroke:#5CB85C,stroke-width:2px;
+linkStyle 7 stroke:#5CB85C,stroke-width:2px;
+linkStyle 8 stroke:#5CB85C,stroke-width:2px;
+linkStyle 9 stroke:#5CB85C,stroke-width:2px;
+linkStyle 10 stroke:#5CB85C,stroke-width:2px;
+linkStyle 11 stroke:#5CB85C,stroke-width:2px;
+
+linkStyle 12 stroke:#8C8C8C,stroke-width:2px;
+linkStyle 13 stroke:#8C8C8C,stroke-width:2px;
+```
+
+### Operating procedure
+
+#### 1. Prepare XCP-ng hosts
+1. On one of the host servers, make sure that the multipath.conf configuration includes your storage equipment.
+   
+   This can be found in the file `/etc/multipath.xenserver/multipath.conf`
+2. If your equipment is not present, ask the manufacturer for the multipath configuration for GNU/Linux otherwise move on to the [next step (Prepare the pool)](../../storage/multipathing/#2-prepare-the-pool)
+   
+   Add it to the file ```/etc/multipath/conf.d/custom.conf```
+
+   :::info
+   In this case, the configuration will be kept after updates.
+   :::
+   
+3. If necessary, migrate the VMs active on the XCP-ng host in question to another one
+4. Reboot the current XCP-ng host on the affected pool
+5. Do the same for all servers in the pool
+
+#### 2. Prepare the pool
+Make sure that multipathing is enabled on the pool. To do this, go to the advanced configuration of the pool
+
+If this is not the case:
+1. Make sure there are **no VMs running** on an iSCSI and/or HBA SR in the pool
+2. Activate "Enable multipathing for all hosts"
+
+#### 3. Configure the SR
+Proceed with the iSCSI SR configuration as indicated in the [storage documentation](../../storage/#iscsi).
+
+## Fiber Channel (HBA)
+### Requirements
+* Check that the Fiber Channel cards model(s) is supported via the HCL
+* Two different Fiber Channel cards (It can be the same model)
```

Comment:
this isn't necessarily required, quite a lot of customers have a single FC card with 2 or 4 FC ports, which are connected to redundant paths on their storage network